src/app/content/highlights/reducer.ts

```diff
@@ -21,8 +21,14 @@ import {
 import { findHighlight } from './utils/reducerUtils';
 import updateSummaryFilters from './utils/updateSummaryFilters';
 
+const initialConfirmationModalState: State['confirmationModal'] = {
```

Comment:
I think this should be somewhere higher in the structure

Reply:
What do you mean? Like put it from:
```
content: {
   ...,
   highlights: {
      ...,
      confirmationModal: {...}
   }
}
```
to
```
content: {
   ...,
   highlights: {...},
   confirmationModal: {...}
}
```
or even higher in state structure?

Reply:
you've put <ConfirmationModal> into the <Content> component so I think the second example would be accurate